lib/extended_color_spinor_utilities.cu

```diff
@@ -486,6 +486,16 @@ namespace quda {
       }else{
         errorQuda("Unsupported Precision %d", src.Precision());
       }
+    } else if (dst.Precision() == QUDA_QUARTER_PRECISION){
+      if(src.Precision() == QUDA_DOUBLE_PRECISION){
+        CopyExtendedColorSpinor(dst, src, parity, location, static_cast<char*>(Dst), static_cast<double*>(Src), static_cast<float*>(dstNorm), 0);
+      }else if(src.Precision() == QUDA_SINGLE_PRECISION){
+        CopyExtendedColorSpinor(dst, src, parity, location, static_cast<char*>(Dst), static_cast<float*>(Src), static_cast<float*>(dstNorm), 0);
+      }else if(src.Precision() == QUDA_HALF_PRECISION){
+        CopyExtendedColorSpinor(dst, src, parity, location, static_cast<char*>(Dst), static_cast<short*>(Src), static_cast<float*>(dstNorm), static_cast<float*>(srcNorm));
+      }else{
```

Comment:
Is there a reason copying quarter -> quarter results in an `errorQuda`?

Reply:
This part of the code is not used by this PR and is commented out entirely. It shows diff because of historical reasons.

@maddyscientist does it make sense to simply remove this code?

Reply:
A proper treatment of extended color-spinor fields is on my TODO list.  If this file is taking up actual compilation time and/or is a non-trivial binary size, then I would suggest commenting out these instantiations and just leaving an error message regardless of precision.